src/plugins/ScrollContainer.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ScrollContainer
+ * @typechecks
+ */
+
+import React from 'react';
+import Scrollbar from 'Scrollbar';
+import PropTypes from 'prop-types';
+import joinClasses from 'joinClasses';
+import cx from 'cx';
+import isEmpty from 'lodash/isEmpty'
+
+class ScrollContainer extends React.Component {
+    state = {};
+
+    _onScrollBarsUpdate = (args) => {
+        this.setState(args);
+    };
+
+    _onVerticalScroll = (/*number*/ scrollPos) => {
+        if (this.state.scrollToY) {
```

Comment:
Will this work if user scrolls to value 0 ?
I think it's better to check for `undefined` instead (if the check is required at all).

(similar for `_onHorizontalScroll`)

Reply:
will take care of this

Reply:
Actually we are checking `this.state.scrollToY` and not `scrollPos`, this should be fine but still took care of comparing with `undefined`